### PR TITLE
Ansible Sanity Tests Updates

### PR DESCRIPTION
**Pull request by JuanDCardozo**

### CODE REVIEW

File: plugins/modules/split.py [hunk 1]
```diff
@@ -1,12 +1,71 @@
...
+  sample: "Incident split successfully."
+"""
 
 from ansible.module_utils.basic import AnsibleModule
+from requests import HTTPError
+import requests
 
 
 def main():
+
```
Comment (cjcrist) [inline plugins/modules/split.py:68]: I'd remove the blank line.

Comment (JuanDCardozo) [reply plugins/modules/split.py:68]: The linting was complaining about not having a blank line

File: plugins/modules/split.py [hunk 2]
```diff
@@ -1,12 +1,71 @@
...
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
-import requests
-from requests import HTTPError
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+
```
Comment (cjcrist) [inline plugins/modules/split.py:10]: extra line

**Review by cjcrist — APPROVED** [review]
I left a couple of minor comments.   Looks good.

**Pull request merged**